compiler/optimizer/OSRDefAnalysis.cpp

```diff
@@ -899,12 +1094,27 @@ int32_t TR_OSRLiveRangeAnalysis::perform()
             //
             _liveVars->empty();
             TR::Block *succ;
-            for (auto edge = block->getSuccessors().begin(); edge != block->getSuccessors().end(); ++edge)
+            auto edge = block->getSuccessors().begin();
+            for (; edge != block->getSuccessors().end(); ++edge)
```

Comment:
I'd consider a while loop here...

Reply:
Both loops have been change to while loops.